src/VTK/log.h

```diff
@@ -0,0 +1,191 @@
+#ifndef __LOG_H__
```

Comment:
license?

Reply:
Actually, this is the only remark which is a blocking one.

Reply:
delete log.h